test/evergreen/doc_update.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#
+# This script attempts to generate Github Apps token and use it to push a local commit to remote.
```

Comment:
I'm glad you asked it. The main consideration is to avoid exposing the credentials - script args feel more likely to be exposed than env variables. Exposure of the former is susceptible to logging lines, and that of the latter needs access to the building host.